setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# W391 blank line at end of file
+# E302 expected 2 blank lines, found 1
+ignore = W391,E302
```

Comment:
I don't understand why the code base doesn't apply E302. It's so much clearer with two blank lines between classes/functions and the ability to add a blank line within a function.